src/Microsoft.AspNet.Mvc.Localization/LocalizedHtmlString.cs

```diff
@@ -26,13 +27,28 @@ public LocalizedHtmlString(string key, string value)
         /// <param name="key">The name of the string resource.</param>
         /// <param name="value">The string resource.</param>
         /// <param name="isResourceNotFound">A flag that indicates if the resource is not found.</param>
-        public LocalizedHtmlString(string key, string value, bool isResourceNotFound)
-            : base(value)
+        /// <param name="arguments">The values to format the <paramref name="value"/> with.</param>
+        public LocalizedHtmlString(string key, string value, bool isResourceNotFound, params object[] arguments)
```

Comment:
You should still provide the constructor without `params` here. The array is created by the callee.